app/schema.graphql

```diff
@@ -1688,6 +1688,11 @@ type Query {
     """HDBSCAN cluster selection epsilon"""
     clusterSelectionEpsilon: Float! = 0
   ): [Cluster!]!
+
+  """
+  The allocated storage capacity of the database (in gibbibytes). Return None if this information is unavailable. 1 gibbibyte is 2^30 bytes.
+  """
+  dbStorageCapacity: Float
```

Comment:
Why not put the units in the variable? - It's kinda confusing because the units for the db tables is in bytes so the same conversion logic can't be used.